### ISSUE TITLE
CRITICAL: Notice and Modal at the same time results in " Uncaught Error: NOT_FOUND_ERR: DOM Exception 8" when using "Notice" as title of mBox.Modal

### ISSUE THREAD

**Issue by philipp-winterle** [stephanwagner/mBox#3]
If you initialize a mBox.Notice and very short after this a mBox.Modal with different contents you will get a  Uncaught Error: NOT_FOUND_ERR: DOM Exception 8. 

I debugged this and got the following information:
- in load(content, title, footer) the first thing happening is setContent(content). everthing fine until here
- next would be this.setTitle(title) which would normally run through the

```
if(content != null && !this.titleContainer) {
```

without any problems. But because you started a notice before the modal window the titleContainer is already filled so that !this.titleContainer == true
- this results in going to this.setContent(content, "title") again and there the shit happens. 

```
if($(content) || $$('.' + content).length > 0) {
                this[where || 'content'].grab($(content) || $$('.' + content)); 
                if($(content)) $(content).setStyle('display', '');
            } else if(content != null) {
                this[where || 'content'].set('html', content);
            }
```

Due to the fact, that the notice has filled our instance of mBox.Core with a Title and a TitleContainer it resolves in $$('.' + content) is filled with an element. But when trying to execute the next line the element or instance or whatever seems to be removed and can no longer be accessed.

**Comment by philipp-winterle** [comment]
Seems like it depends on which title you give the modal window. In my case it is called "Notice". After some testing with "NoticeAAA" it wasnt possible to reproduce it for now. 

```
this[where || 'content'].grab($(content) || $$('.' + content));
```

In this case content would be "Notice" and will be checked for a Element with that ID or Class. In this case one or more elements with the class "Notice" max exists because a notice is floating. Thats why the code goes this way and not the right one.

```
if($(content) || $$('.' + content).length > 0) {
```

This shouldnt be allowed to be true for the case "Notice"

**Comment by stephanwagner** [comment]
You are right! This shouldn't be allowed! Ill still leave this one open and hopefully ill find some time to fix this in 0.2.7, my Idea was anyways to rename the classes .Notice .Core .Modal etc. to .mBoxNotice. This is much saver.

**Comment by philipp-winterle** [comment]
Godd idea. Would reduce the chance of getting that issue at any way.
